forge/db/models/Team.js

```diff
@@ -559,6 +578,50 @@ module.exports = {
                     await this.setTeamType(teamType)
                     await this.save()
                     await this.reload({ include: [{ model: M.TeamType }] })
+                },
+
+                /**
+                 * Suspend the team
+                 *  - sets suspended=true
+                 *  - suspends all team instances
+                 *
+                 * When running with EE, this function is replaced via ee/lib/billing/Team.js
+                 * to complete billing related operations first
+                 */
+                suspend: async function () {
+                    this.suspended = true
+                    await this.save()
+                    // get list of running Instances
+                    const instanceList = await app.db.models.Project.findAll({
+                        attributes: [
+                            'id',
+                            'state',
+                            'ProjectStackId',
+                            'TeamId'
+                        ],
+                        where: {
+                            TeamId: this.id,
+                            state: {
+                                [Op.eq]: 'running'
```

Comment:
This could miss "starting" instances.

Reply:
`starting` is an inflight state that never hits the database. Within the database, the states are either 'running' or 'suspended'